### ISSUE TITLE
[Feature request]: Being able to change light/dark theme in project menu

### ISSUE THREAD

**Issue by louisgreiner** [SchweizerischeBundesbahnen/netzgrafik-editor-frontend#241]
### Preflight Checklist

- [X] I have read the [Contributing Guidelines](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/CONTRIBUTING.md) for this project.
- [X] I agree to follow the [Code of Conduct](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issue tracker](https://github.com/orgs/sbb-design-systems/projects/1) for an issue that matches the one I want to file, without success.

### Request type

Request for enhancement of a component

### Functionality

![image](https://github.com/user-attachments/assets/eca6335e-0054-4c2b-9eb1-412845a4ae93)

It would be nice to be able to change the theme (light/dark) directly in project menu, not to have to wait to be in a variant

**Comment by aiAdrian** [comment]
I totally agree that we need to consider whether and how we provide all settings (global - i.e., valid across all projects/variants and user-dependent (Browser: local storage)) to the user. The color scheme is basically needed only in the editor, like all other settings, i.e., project and variant pages are 'colorless.' Here, only the Dark/Light mode is reacted to. Therefore, I wonder whether it should be handled like the language setting or if we should leave the settings as they are currently. Then we would just need to find a logic for how we can switch between Dark/Light modes.
I would appreciate it if we could build it analogously to the language setting.

=> These adjustments are not yet perfectly designed. Perhaps we could discuss this with a UX expert, or they could make a suggestion.

=> To ***summarize***: We have global settings like language, which must take effect from the first page, colors, Editor: travel time, Route graphics: temporal scaling which only take effect in the editor or in the route graphics. These settings should be made as accessible as possible to the user so that they can find these functions quickly and easily

![image](https://github.com/user-attachments/assets/b7c6332c-31f0-4017-b5c3-2555c28a7034)
